docker-entrypoint.sh

```diff
@@ -6,35 +6,11 @@ WORKER_CLASS=${WORKER_CLASS:-gevent}
 ACCESS_LOG=${ACCESS_LOG:--}
 ERROR_LOG=${ERROR_LOG:--}
 WORKER_TEMP_DIR=${WORKER_TEMP_DIR:-/dev/shm}
-SECRET_KEY=${SECRET_KEY:-}
-DATABASE_URL=${DATABASE_URL:-}
 
-# Check that a .ctfd_secret_key file or SECRET_KEY envvar is set
-if [ ! -f .ctfd_secret_key ] && [ -z "$SECRET_KEY" ]; then
-    if [ $WORKERS -gt 1 ]; then
-        echo "[ ERROR ] You are configured to use more than 1 worker."
-        echo "[ ERROR ] To do this, you must define the SECRET_KEY environment variable or create a .ctfd_secret_key file."
-        echo "[ ERROR ] Exiting..."
-        exit 1
-    fi
-fi
```

Comment:
I think it's a bad idea to remove this. You're right that the script doesn't consider if config.ini is hard coded but I think with a Docker container the better solution would be to use environment variables anyway.

Reply:
but `config.py` does respects env variables...

Reply:
It respects the envvars but it won't fail the container if the secret key is being randomly generated. This check is primarily for having different secret keys across different workers.

Reply:
I wrote the comments because my script actually imported `config.py`, so it's impossible for different workers to have different secret keys.

Reply:
I still think there are cases that could cause errors that are hard to debug. It's better to leave this as is. I doubt it causes any issues.